tools/conversion/convert_to_tflite.py

```diff
@@ -127,54 +121,67 @@ def convert(backbone_settings, classifier_settings, output_name, plot_model, ver
     export_keras_to_tflite(keras_file, tflite_file)
 
     if fake_weights:
-        logging.error("************************* Warning!! **************************")
-        logging.error("Weights in checkpoint did not match weights required by network")
-        logging.error("Fake weights were generated where they were needed!!!!!!!!!!!!")
-        logging.error("************************* Warning!! **************************")
+        logging.warning(
```

Comment:
This is now a warning again? I think it was an `error` on purpose so that it's not swallowed when the log level is too low.

Reply:
Oh, we can change it back. I was just confused because the `error` log (which to me is something that is actually breaking) started with the word "warning" (which to me is something that is not breaking)

Reply:
As discussed, we should make sure that the default log level includes warnings